src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 ---
 
  * Add `XmlEncoder::SAVE_OPTIONS` context option
+ * Add `SupportedTypesMethodInterface` for normalizers and denormalizers that can provide supported types outside from calls to their `supports*()` methods
```

Comment:
What about just adding `getSupportedTypes()` to Normalizer and DenormalizeInterface, with a BC layer of course?